impeller/typographer/backends/skia/text_frame_skia.cc

```diff
@@ -57,12 +56,6 @@ TextFrame MakeTextFrameFromTextBlobSkia(const sk_sp<SkTextBlob>& blob) {
     const auto glyph_count = run.glyphCount();
     const auto* glyphs = run.glyphs();
     switch (run.positioning()) {
-      case SkTextBlobRunIterator::kDefault_Positioning:
-        FML_DLOG(ERROR) << "Unimplemented.";
-        break;
-      case SkTextBlobRunIterator::kHorizontal_Positioning:
-        FML_DLOG(ERROR) << "Unimplemented.";
-        break;
```

Comment:
Since we're never going to use these, I deleted them and left it for the default.

Reply:
I have a slight preference for removing the default so that we know if Skia adds some other value to this that we could be handling.